admin/app/com/lucidchart/piezo/admin/controllers/TriggerFormHelper.scala

```diff
@@ -167,14 +172,15 @@ class TriggerFormHelper(scheduler: Scheduler, monitoringTeams: MonitoringTeams)
       .verifying(
         "A valid team is required if monitoring is on",
         fields => {
-          !monitoringTeams.teamsDefined || fields.priority == TriggerMonitoringPriority.Off || fields.monitoringTeam.exists(monitoringTeams.value.contains[String])
-        }
+          !monitoringTeams.teamsDefined || fields.priority == TriggerMonitoringPriority.Off || fields.monitoringTeam
+            .exists(monitoringTeams.value.contains[String])
+        },
       ),
   )
 }
 
 object MaxSecondsBetweenSuccessesFormatter extends Formatter[Int] {
-  override val format = Some(("format.triggerMaxErrorTime", Nil))
+  override val format: Option[(String, Seq[Nothing])] = Some(("format.triggerMaxErrorTime", Nil))
```

Comment:
Nit: Should we stick to original signature of `format`? I.e. `Option[(String, Seq[Any])]`:
https://www.playframework.com/documentation/2.9.3/api/scala/play/api/data/format/Formatter.html#format:Option[(String,Seq[Any])]